celery_with_django/django_celery_project/settings.py

```diff
@@ -0,0 +1,172 @@
+"""
+Django settings for django_celery_project project.
+
+Generated by 'django-admin startproject' using Django 5.0.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+from pathlib import Path
+import os
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-w63dld2gxku7hkplgs%2i4hj3f0t_t8vrxxx0x6$u=pcq=g#7u'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
```

Comment:
Will this allow localhost by default? Also when we push the code or deploy the Debug will be `False`, I think in that case you need to allow the hosts, For now this is fine. Just keep note

Reply:
ok, I keep this note